hass_nabucasa/__init__.py

```diff
@@ -40,7 +40,7 @@ class Cloud(Generic[_ClientT]):
     def __init__(
         self,
         client: _ClientT,
-        mode: str,
+        mode: Literal["development", "production"],
```

Comment:
```suggestion
        mode: Literal[MODE_DEV, MODE_PROD],
```

Reply:
`MODE_PROD` needs to be imported from `.const`, but GitHub does not allow me to suggest that.

Reply:
No harm, because what you're suggesting is not valid: https://stackoverflow.com/questions/76382022/create-literal-type-from-constant-variables

Reply:
Thats just stupid...